resources.txt

```diff
@@ -107,3 +107,7 @@ https://api.sberbank.ru
 https://md.mos.ru
 https://dsszrc.ru
 https://www.moex.com
+https://fsb.ru
+https://yandex.ru
+https://mail.ru
+https://avito.ru
```

Comment:
avito is already exist

Reply:
will not yandex or mail, this target is too huge for chaos attack yet

added fsb in other commit

thanks for help anyway